shared/src/commonMain/kotlin/vn/luongvo/kmm/survey/data/remote/body/LoginRequestBody.kt

```diff
@@ -0,0 +1,21 @@
+package vn.luongvo.kmm.survey.data.remote.body
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import vn.luongvo.kmm.survey.BuildKonfig
+
+private const val GRANT_TYPE_PASSWORD = "password"
```

Comment:
Curious can we define this as `LoginRequestBody`'s companion object?

Reply:
@blyscuit There are several ways to define const in Kotlin: companion object or top-level const. The Android team prefers top-level const https://nimblehq.co/compass/development/code-conventions/kotlin/#constants